amm/drink-tests/src/stable_swap_tests/tests_rated.rs

```diff
@@ -0,0 +1,557 @@
+use crate::mock_sazero_rate_contract;
+use crate::stable_pool_contract;
+use crate::utils::*;
+
+use super::*;
+
+use drink::{self, runtime::MinimalRuntime, session::Session};
+use ink_primitives::AccountId;
+use ink_wrapper_types::{Connection, ToAccountId};
+
+const WAZERO_DEC: u8 = 12;
+const SAZERO_DEC: u8 = 12;
+
+const ONE_LPT: u128 = 10u128.pow(18);
+const ONE_WAZERO: u128 = 10u128.pow(WAZERO_DEC as u32);
+const ONE_SAZERO: u128 = 10u128.pow(SAZERO_DEC as u32);
+
+const EXPIRE_TS: u64 = 24 * 3600 * 1000; // 24h
```

Comment:
What `TS` is supposed to mean? I see it's in millis so maybe `EXPIRE_TIME_MILLIS`?